### PR TITLE
use local storage-type for content cluster resources

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

File: vespa-cloud/cord-19-search/src/main/application/services.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
       <document-processing cluster="default" chain="bert-tensorizer" />
     </documents>
     <nodes count="[2,4]">
-      <resources memory="8Gb" vcpu="4" disk="50Gb" disk-speed="fast"/>
+      <resources memory="8Gb" vcpu="4" disk="50Gb" storage-type="local" />
```
Comment (bratseth) [inline vespa-cloud/cord-19-search/src/main/application/services.xml:64]: I don't think this flavor exists?

File: vespa-cloud/search-suggestion/src/main/application/services.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
             <document mode="streaming" type="term"/>
         </documents>
         <nodes count="[2,3]">
-            <resources vcpu="2.0" memory="8Gb" disk="50Gb"/>
+            <resources vcpu="2.0" memory="8Gb" disk="50Gb" storage-type="local" />
```
Comment (bratseth) [inline vespa-cloud/search-suggestion/src/main/application/services.xml:41]: Are you sure this flavor exists?

**Comment by kkraune** [comment]
> Are you sure this flavor exists?

Well, who knows? It is hard to say from https://cloud.vespa.ai/en/reference/services, so I want to improve this by pasting the error message I most likely will get from deploying this. This is a bit under-documented, so lets dogfood based on what the users will see. There will be more PRs ....

**Comment by bratseth** [comment]
Fine.

**Review by bratseth — APPROVED** [review]

**Pull request merged**